google-cloud-testing/storage-benchwrapper/src/main/java/com/benchwrapper/StorageBenchWrapperImpl.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.benchwrapper;
+
+import io.grpc.stub.StreamObserver;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+
+import com.benchwrapper.StorageBenchWrapperGrpc.StorageBenchWrapperImplBase;
+import com.google.cloud.ReadChannel;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageException;
+import com.google.cloud.storage.StorageOptions;
+
+class StorageBenchWrapperImpl extends StorageBenchWrapperImplBase {
+  private Storage client;
+
+  public StorageBenchWrapperImpl() {
+    client = StorageOptions.newBuilder()
+        .setHost("http://localhost:8080")
```

Comment:
What's running locally on port 8080? A proxy to GCS?

Reply:
Yes. I'd actually like to change the port to a flag: in other languages we have `--port` defined. To do so, what's the best approach? Something like http://commons.apache.org/proper/commons-cli/?

Reply:
System properties or environment variables would probably make sense. With System properties, you would add `-Dpropname=propvalue` to your Maven command or configure through your IDE.

Reply:
Done